servers/physics_2d/godot_space_2d.cpp

```diff
@@ -533,20 +533,14 @@ int GodotSpace2D::_cull_aabb_for_body(GodotBody2D *p_body, const Rect2 &p_aabb)
 }
 
 bool GodotSpace2D::test_body_motion(GodotBody2D *p_body, const PhysicsServer2D::MotionParameters &p_parameters, PhysicsServer2D::MotionResult *r_result) {
-	//give me back regular physics engine logic
-	//this is madness
-	//and most people using this function will think
-	//what it does is simpler than using physics
-	//this took about a week to get right..
-	//but is it right? who knows at this point..
-
+	
+	TestBodyMotionData tbmd;
```

Comment:
```suggestion
	TestBodyMotionData test_data;
```
Or similar, we avoid abbreviated and vague names like this